src/store/index.ts

```diff
@@ -103,6 +105,25 @@ function sortData(data: Array<Region>, dataType: DataType) {
   });
 }
 
+function addSearchTokensForLevel(tokens: Set<string>, region: Region, level: TaxonomyType): void {
```

Comment:
can we remove the handling/creation of "shortName"? seems like it's only the for the us and right now, somewhat confusingly, searching for "usa" hides the united states because searchTokens are generated only from country's names and codes